libpsautohint/include/psautohint.h

```diff
@@ -134,13 +133,12 @@ ACLIB_API void AC_SetReportRetryCB(AC_RETRYPTR retryCB);
  * pointer to null terminated C string containing fontinfo for the bez glyph.
  *
  * Hint information is added to the bez data and returned to the caller through
- * the buffer dstbezdata. dstbezdata must be allocated before the call and a
+ * the buffer dstbezdata. dstbezdata should be allocated before the call and a
```

Comment:
Why did you change the wording from `must` to `should` here? Is there a scenario in which it's OK to not allocate it before the call? If so, it might be helpful to describe that scenario.

Reply:
Since we now re-allocate the output buffer if it is too small, not allocating it and passing 0 for the length should work. But actually, after checking that, it doesn’t, I’ll revert the wording here.